apps/frontend/web/src/pages/Overview.tsx

```diff
@@ -1,10 +1,52 @@
-import React from 'react';
-import { Box, Image, Text } from '@chakra-ui/react';
+import { Button, Box, Image, Text } from '@chakra-ui/react';
+import { useState } from 'react';
 
 const Overview = () => {
+  const [showGetCheck, setShowGetCheck] = useState(true);
```

Comment:
C'est une bonne pratique de préfixer les boolean avec de l'impératif : `should` `is` `must`